db/migrate/20200607110209_fix_latest_flag_for_individual_target_submissions.rb

```diff
@@ -0,0 +1,47 @@
+class FixLatestFlagForIndividualTargetSubmissions < ActiveRecord::Migration[6.0]
+  class TimelineEvent < ApplicationRecord
+    has_many :timeline_event_owners
+    belongs_to :target
+    has_many :target_evaluation_criteria, through: :target
+  end
+
+  class TimelineEventOwner < ApplicationRecord
+    belongs_to :timeline_event
+    belongs_to :founder
+  end
+
+  class Founder < ApplicationRecord
+    has_many :timeline_event_owners
+    has_many :timeline_events, through: :timeline_event_owners
+  end
+
+  class Target < ApplicationRecord
+    ROLE_TEAM = 'team'
+
+    has_many :target_evaluation_criteria
+
+    def team_target?
+      role == ROLE_TEAM
+    end
```

Comment:
We don't really need the constant or the method in this migration. See modification to usage below.

```suggestion
    has_many :target_evaluation_criteria
```